### DISCUSSION TITLE
Choose tag in noticeType 7

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#397] [Q&A] [answered]
Hi Labs!
in notice subType 7 there 2 fields that differ only for the xpath Predicate 
BT-765-Lot[repeatable=false,xpathRelative=cac:ContractingSystem/cbc:ContractingSystemTypeCode[@listName='framework-agreement']]
BT-766-Lot[repeatable=false,xpathRelative=cac:ContractingSystem/cbc:ContractingSystemTypeCode[@listName='dps-usage']]

you wrote : 
The XPath provided for some nodes in the xmlStructure specifies a predicate (within square brackets). This is because the same XML element may have different semantics depending on some condition. This variation in semantics is significant for the physical structure of the notice (XML structure) and is represented with separate nodes in the xmlStructure list. OK

In the past I only met predicates to find a specific node in existing xml , making notice 7 for example,  how should I use this predicate ?
Thanks a lot !
Renato

**Comment by eduardsmirnov** [comment, marked as answer]
Non-official comment:
BT-765-Lot has only XPath with predicate [@listName='framework-agreement'], so you should add listname to ContractingSystemTypeCode.
 
Predicate [@listName='dps-usage'] goes for BT-766-Lot, you should add another listname attribute there.

You may have both ContractingSystemTypeCode saved at the same time, then listname attribute is what makes them different.

**Comment by rengit62** [comment]
Thanks Eduard !
Renato
